Legion/VaultOfTheWardens/Tormentorum.lua

```diff
@@ -115,3 +115,36 @@ end
 function mod:FrighteningShout(args)
 	self:Message(args.spellId, "Urgent", "Warning", CL.casting:format(args.spellName))
 end
+
+do
+	local prev = 0
+	function mod:GroundEffectDamage(args)
+		local t = GetTime()
+		if self:Me(args.destGUID) and t-prev > 1.5 then
+			prev = t
+			self:Message(args.spellId, "Personal", "Alert", CL.underyou:format(args.spellName))
+		end
+	end
+end
+
+do
+	local prev, list = 0, mod:NewTargetList()
+	function mod:FleshToStoneApplied(args)
+		if args.amount < 7 then
+			return
+		end
+
+		local t = GetTime()   -- 2 of these adds spawn, they will finish their casts
+		if t - prev < 5 then  -- roughly at the same time, throttling to avoid being spammy
+			return
+		end
+		prev = t
+
+		local count = #list+1
+		list[count] = args.destName
+
+		if count == 1 then
+			self:ScheduleTimer("TargetMessage", 0.3, args.spellId, list, "Urgent", self:Dispeller("magic") and "Warning", format("%s (7+)", args.spellName), nil, true)

```

Comment:
I think having a sound for everyone with high stacks and Dispellers would be preferable here (change `sound` to `"Warning"` and `alwaysPlaySound` to `self:Dispeller("magic")`)
Maybe even change the sound depending on stacks? `Alarm` for 7 and 8, `Warning` for 9?

Reply:
Get rid of the `local format` nonsense and use `:format` correctly like the rest of the module.

That being said, the message itself seems... odd. Why not just use a standard count?

Reply:
@funkydude I thought about that but it takes an integer as its second argument. People can have different amount of stacks (for example, if someone used an immunity when they had 1 stack and since then 7 more stacks were applied to the entire party, everyone but that person would have 8), whose stacks should we print? Showing the minimum amount (that makes a message show up) just felt appropriately to me.

Reply:
@elvador changed the code as per your 1st suggestion. I am not really sure how to implement the 2nd suggestion, see the message above this one.